src/Compilers/CSharp/Portable/Binder/AliasAndExternAliasDirective.cs

```diff
@@ -16,12 +14,16 @@ namespace Microsoft.CodeAnalysis.CSharp
     internal struct AliasAndExternAliasDirective
     {
         public readonly AliasSymbol Alias;
-        public readonly ExternAliasDirectiveSyntax ExternAliasDirective;
+        public readonly SyntaxReference? ExternAliasDirectiveReference;
+        public readonly bool SkipInLookup;
```

Comment:
>SkipInLookup [](start = 29, length = 12)

Consider adding a comment (Extern aliases not allowed in interactive submissions, so skipped)

Reply:
> Consider adding a comment (Extern aliases not allowed in interactive submissions, so skipped)

I am not trying to define here a reason why the alias must be skipped in lookup. And the reason you suggest is not it. Right now the only reason is that the alias is a duplicate.

---
In reply to: [588799493](https://github.com/dotnet/roslyn/pull/51676#discussion_r588799493) [](ancestors = 588799493)